envs/monkey_zoo/terraform/firewalls.tf

```diff
@@ -156,6 +156,21 @@ resource "google_compute_firewall" "powershell-48-allow" {
   target_tags = ["powershell-48"]
 }
 
+resource "google_compute_firewall" "block-powershell-48-to-depth-3 " {
+  name    = "${local.resource_prefix}block-powershell-48-to-depth-3 "
+  network = google_compute_network.monkeyzoo.name
+
+  deny {
+    protocol = "all"
+  }
+
+  direction = "INGRESS"
+  priority = "1000"
+  source_tags = ["powershell-48"]
+  target_tags = ["mimikatz-15", "powershell-45", "powershell-47", "tunneling-9"]
```

Comment:
Instead of blacklisting we should whitelist. Create a firewall rule that blocks all outgoing traffic from 48. Then create another firewall rule that allows outgoing traffic from 48 to island and 45. This way we won't need to modify terraform scripts any time depth-3 test suite changes